sbin/prepareWorkspace.sh

```
@@ -179,8 +194,6 @@ checkoutRequiredCodeToBuild() {
           else
             echo "Failed cmd: git reset --hard \"origin/${BUILD_CONFIG[BRANCH]}\""
           fi
-        else
-          echo "Failed cmd: git fetch --all ${BUILD_CONFIG[SHALLOW_CLONE_OPTION]}"
```

Should this have been deleted? fetch --all will fail silently